### PR TITLE
MTP-1152 set up svelte storybook in version 7

### PR DESCRIPTION
**Pull request by lha047**
- using designsystem-css
- use vite and vitest

### CODE REVIEW

**Comment by petterkoding** [comment]
Works for on my part when I tested scss and storybook. Will take a closer look when I have more time :)

**Review by petterkoding — APPROVED** [review]
Looks good :D

**Review by amundlindstol — APPROVED** [review]
LGTM

File: mt-kit/core/svelte/src/lib/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export { default as Published } from './svelte/components/Published.svelte'
+export { default as SummaryDetail } from './svelte/components/SummaryDetail.svelte'
+export { default as Tags } from './svelte/components/Tags.svelte'
+
+export { default as AnimalDisease } from './svelte/content/AnimalDisease.svelte'
+export { default as CurrentIssues } from './svelte/content/CurrentIssues.svelte'
+export { default as HighlightedContentLink } from './svelte/content/HighlightedContentLink.svelte'
+export { default as MenuItems } from './svelte/content/MenuItems.svelte'
+export { default as MenuPointsIcons } from './svelte/content/MenuPointsIcons.svelte'
+export { default as Status } from './svelte/content/Status.svelte'
```
Comment (TheBoosja) [inline mt-kit/core/svelte/src/lib/index.ts:41]: Don't know that much about it, but does this support tree-shaking? If we want to do something like `import Status from '@mattilsynet/design-system/status'` (don't quite remember the package name :slightly_smiling_face:)

**Review by TheBoosja — APPROVED** [review]
Looks good

**Pull request merged**